# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webcodecs/WebCodecsUtils.h
# Commit: 2e334c8a1160
# Full Hash: 2e334c8a11602d2783fd894a237f170267b1fa94
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 3
# Description:
#   Bug 1749047 - Allow printing a VideoDecoderConfig and a VideoDecoderConfigInternal. r=chunmin
#   
#   This is very useful to understand in the log what config is output in what
#   encoder output callback, with respect to configure(...) calls.
#   
# ==============================================================================

diff -r be191f9ac293 -r 2e334c8a1160 dom/media/webcodecs/WebCodecsUtils.h
--- a/dom/media/webcodecs/WebCodecsUtils.h	Fri Dec 22 21:39:13 2023 +0000
+++ b/dom/media/webcodecs/WebCodecsUtils.h	Fri Dec 22 21:39:14 2023 +0000
@@ -227,6 +227,8 @@
 
 Maybe<CodecType> CodecStringToCodecType(const nsAString& aCodecString);
 
+nsString ConfigToString(const VideoDecoderConfig& aConfig);
+
 } // namespace dom
 
 }  // namespace mozilla
